.github/workflows/build.yaml

```diff
@@ -71,34 +71,30 @@ jobs:
         uses: azure/setup-kubectl@v3
         with:
           version: v1.21.2
-      - 
-        name: Create KUBECONFIG
-        env:
-          KUBE_CONFIG: '${{ secrets.KUBECONFIG_TEST }}'
-        run: |
-          mkdir -p $HOME/.kube
-          echo "$KUBE_CONFIG"
-          echo "$KUBE_CONFIG" > $HOME/.kube/config
       - 
         name: Deploy Helm Release
         env:
           TRAINING_HELM_RELEASE: 'pr-${{ github.event.pull_request.number }}'
-          TRAINING_NAMESPACE: 'pitc-ansible-training-test'
+          TRAINING_NAMESPACE: 'pitc-ansible-training-stg'
           TRAINING_VERSION: '${{ github.sha }}'
+          KUBE_TOKEN: '${{ secrets.OPENSHIFT_SA_TOKEN_STG }}'
+          KUBE_URL: '${{ secrets.OPENSHIFT_API_URL }}'
         run: |
-          helm upgrade $TRAINING_HELM_RELEASE acend-training-chart --install --wait --kubeconfig $HOME/.kube/config  --namespace=$TRAINING_NAMESPACE --set=app.name=$HELM_RELEASE --set=app.version=$TRAINING_VERSION --repo=https://acend.github.io/helm-charts/ --values=helm-chart/values.yaml --atomic
+          helm upgrade $TRAINING_HELM_RELEASE acend-training-chart --debug --install --wait --kube-apiserver=$KUBE_URL --kube-token=$KUBE_TOKEN --namespace=$TRAINING_NAMESPACE --set=app.name=$HELM_RELEASE --set=app.version=$TRAINING_VERSION --repo=https://acend.github.io/helm-charts/ --values=helm-chart/values_openshift.yaml --atomic
       - 
         name: Redeploy Deployments
         env:
           TRAINING_HELM_RELEASE: 'pr-${{ github.event.pull_request.number }}'
-          TRAINING_NAMESPACE: 'pitc-ansible-training-test'
+          TRAINING_NAMESPACE: 'pitc-ansible-training-stg'
+          KUBE_TOKEN: '${{ secrets.OPENSHIFT_SA_TOKEN_STG }}'
+          KUBE_URL: '${{ secrets.OPENSHIFT_API_URL }}'
         run: |
-          kubectl rollout restart deployment/$TRAINING_HELM_RELEASE-ansible-puzzle --kubeconfig $HOME/.kube/config --namespace $TRAINING_NAMESPACE
+          kubectl rollout restart deployment/$TRAINING_HELM_RELEASE-ansible-puzzle --server=$KUBE_URL --token=$KUBE_TOKEN --namespace $TRAINING_NAMESPACE
       - 
         name: Comment on PR
         uses: marocchino/sticky-pull-request-comment@v2
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           message: |
            PR Environments: 
-           *  puzzle <https://ansible-pr-${{ github.event.pull_request.number }}.k8s.puzzle.ch>
+           *  puzzle <https://ansible-pr-${{ github.event.pull_request.number }}.puzzle.ch>
```

Comment:
are those dns-entry get created automagicly?

Reply:
Hmmm, looks like this needs some more tweaking, currently the Ingress is created but not the route, which should automatically be created by OPenshift.

Reply:
the acend helm chart was broken in the meantime, so it created the previous non-functional ingress again, should be fixed now.

Reply:
works now. the PR links a la  `ansible-pr-XYZ.ocp.cloudscale.puzzle.ch` are handled by Openshift